### PR TITLE
Update Office 2021 for the article

### PR DESCRIPTION
**Pull request by abhinav2605**

### CODE REVIEW

File: docs/vsto/visual-studio-tools-for-office-runtime.md [hunk 1]
```diff
@@ -43,6 +43,6 @@ The following is a list of the VSTO Runtime releases and the corresponding versi
 | 10.0.60715 | 07/20/2015 | Microsoft Office 2016 |
 | 10.0.60724 | 07/24/2015 | Microsoft Office 2016 |
 | 10.0.60825 | 11/19/2016 | Microsoft Office 2016 |
-| 10.0.60828 | 01/12/2018 | Microsoft Office 2019 / <br>Office 365 |
+| 10.0.60828 | 01/12/2018 | Microsoft Office 2019 / <br>Office 365 /| <br>Office 2021 |
```
Comment (John-Hart) [inline docs/vsto/visual-studio-tools-for-office-runtime.md:46]: I don't think you want the vertical bar between Office 365 and Office 2021 as that would try and add another column

**Review by John-Hart — CHANGES_REQUESTED** [review]

**Comment by jborsecnik** [comment]
@John-Hart, 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/vsto/visual-studio-tools-for-office-runtime.md (hunk 1 above)
Comment (j-martens) [reply docs/vsto/visual-studio-tools-for-office-runtime.md:46]: @abhinav2605  can you make the correction please?

Comment (abhinav2605) [reply docs/vsto/visual-studio-tools-for-office-runtime.md]: I have updated the change for vertical bar(pipe).

**Pull request merged**